microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -233,15 +332,6 @@ def _install_dependencies(self) -> None:
             # TODO: Workaround for type checker below
             node.os.install_packages(list(self._redhat_packages))
 
-            if not self.is_connect_x3:
-                self.__execute_assert_zero(
-                    f"dracut --add-drivers '{mellanox_drivers} ib_uverbs' -f", cwd
-                )
-            self.__execute_assert_zero(
-                "modprobe -a ib_core ib_uverbs mlx4_en mlx4_core mlx5_core mlx4_ib"
-                + " mlx5_ib",
-                cwd,
-            )  # add mellanox drivers
             self.__execute_assert_zero("systemctl enable rdma", cwd)
```

Comment:
Let's replace them with execute and expected_exit_code pattern.

Reply:
I agree but let's do it in a new PR